src/pages/Dashboard.js

```diff
@@ -286,85 +300,124 @@ const DashBoardComponent = () => {
       </FirstRow>
 
       {/* Doctors View - Patients List along with current conditions */}
-      <PatientsWrapper isInitLoading={isInitLoading}>
-        {patients?.length > 0
-          ? patients?.map((patient) => (
-              <PatientCard key={patient.patientId} patient={patient} />
-            ))
-          : searchText?.length > 0 && (
-              <NoPatientsWrapper>
-                <p>
-                  <strong>No results found</strong>
-                </p>
-                <Button onClick={clearSearchInput} className="link-button">
-                  Back to dashboard
-                </Button>
-              </NoPatientsWrapper>
+      {isMobile ? (
+        <PatientsWrapper isInitLoading={isInitLoading}>
+          {patients?.length > 0
+            ? patients?.map((patient) => (
+                <PatientCard key={patient.patientId} patient={patient} />
+              ))
+            : searchText?.length > 0 && (
+                <NoPatientsWrapper>
+                  <p>
+                    <strong>No results found</strong>
+                  </p>
+                  <Button onClick={clearSearchInput} className="link-button">
+                    Back to dashboard
+                  </Button>
+                </NoPatientsWrapper>
+              )}
+          {isInitLoading && <SpinnerComponent isFullScreen={false} />}
+          {hasNext ? (
+            <div className="load-more-container m-3">
+              <Button
+                onClick={incrementPage}
+                disabled={isInitLoading || isShowSearchSpinner}
+                className="btn-load-more btn btn-covin">
+                {isInitLoading || isShowSearchSpinner ? (
+                  <div className="lds-spinner">
+                    {[...Array(12).keys()].map((i) => (
+                      <span key={i} />
+                    ))}
+                  </div>
+                ) : (
+                  <>Load More</>
+                )}
+              </Button>
+            </div>
+          ) : null}
+        </PatientsWrapper>
+      ) : (
+        <DeskTopViewPatient isInitLoading={isInitLoading}>
+          <HeaderSearchWrap className="w-100 mb-3">
+            {!isLightVersion && (
```

Comment:
I think we can get rid of isLightVersion everywhere

Reply:
this might be required in the future so will leave it as it is for now